core/src/test/java/com/datastax/oss/driver/internal/core/data/AccessibleByIdTestBase.java

```diff
@@ -468,14 +468,15 @@ public void should_get_with_explicit_codec_by_name() {
     assertThat(s).isEqualTo("1");
   }
 
+  @SuppressWarnings("UnusedAssignment")
```

Comment:
If you don't re-assign you get a compile error, so the only alternative here was to suppress the unused assignment warning.

Reply:
It's a good thing that the error is issued by the ErrorProne compiler, client applications are free to use it or not depending on the level of check that they desire.